css/bulma.css

```diff
@@ -5049,10 +5044,7 @@ button.dropdown-item.is-active {
 
 .message.is-dark .message-body {
   border-color: #363636;
-}
-
-.message.is-primary {
-  background-color: #ebfffc;
```

Comment:
The background color of a message applies only to the message body, so it was moved there as a small optimization